demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -334,28 +335,28 @@ def copy_packs_content_to_packs_bundle(self, packs):
                 os.mkdir(dest_dir)
                 if dir_name in DIR_TO_PREFIX:
                     packages_dirs = get_child_directories(content_dir)
-                    for package_dir in packages_dirs:
-                        ymls, _ = get_yml_paths_in_dir(package_dir, error_msg='')
-                        if not ymls or (len(ymls) == 1 and ymls[0].endswith('_unified.yml')):
-                            msg = 'Skipping package: {} -'.format(package_dir)
-                            if not ymls:
-                                print_warning('{} No yml files found in the package directory'.format(msg))
-                            else:
-                                print_warning('{} Only unified yml found in the package directory'.format(msg))
-                            continue
-                        package_dir_name = os.path.basename(package_dir)
-                        unifier = Unifier(package_dir, dir_name, dest_dir)
-                        unifier.merge_script_package_to_yml()
-
-                        # also copy CHANGELOG markdown files over (should only be one per package)
-                        package_files = get_child_files(package_dir)
-                        changelog_files = [
-                            file_path
-                            for file_path in package_files if 'CHANGELOG.md' in file_path
-                        ]
-                        for md_file_path in changelog_files:
-                            md_out_name = '{}-{}_CHANGELOG.md'.format(DIR_TO_PREFIX.get(dir_name), package_dir_name)
-                            shutil.copyfile(md_file_path, os.path.join(dest_dir, md_out_name))
+
+                    if packages_dirs:  # split yml files directories
+                        for package_dir in packages_dirs:
+                            ymls, _ = get_yml_paths_in_dir(package_dir, error_msg='')
+                            if not ymls or (len(ymls) == 1 and ymls[0].endswith('_unified.yml')):
+                                msg = 'Skipping package: {} -'.format(package_dir)
+                                if not ymls:
+                                    print_warning('{} No yml files found in the package directory'.format(msg))
+                                else:
+                                    print_warning('{} Only unified yml found in the package directory'.format(msg))
+                                continue
+                            unifier = Unifier(package_dir, dir_name, dest_dir)
+                            unifier.merge_script_package_to_yml()
+
+                    non_split_yml_files = [f for f in os.listdir(content_dir)
+                                           if os.path.isfile(os.path.join(content_dir, f)) and
+                                           (fnmatch.fnmatch(f, 'integration-*.yml') or
```

Comment:
are we sure we need the prefix in the packs?

Reply:
Take a look at Legacy pack.

Reply:
but is it still mandatory?
it's not mandatory for playbooks, why should it be for other entities?
we can continue this discussion later